index.html

```diff
@@ -2,108 +2,91 @@
-          <h3>Torazer</h3>
-          <span class="tag is-warning">Moderator</span>
-        </div>
-
-        <div class="column">
-          <img src="/img/people/trashy.jpg" class="someone mod">
-          <h3>TrashyClassy</h3>
-          <span class="tag is-warning">Moderator</span>
-        </div>
-        
-        <div class="column">
-          <a href="https://www.youtube.com/channel/UC04awUMPfLwEGtCCWvYq-UQ" target="_blank" rel="noopener noreferrer"><img src="/img/people/anidox.jpg" class="someone mod"></a>
-          <h3>Anidox</h3>
-          <span class="tag is-warning">Moderator</span>
-        </div>
-		
-		<div class="column">
-          <a href="https://twitter.com/SonicHackYT" target="_blank" rel="noopener noreferrer"><img src="/img/people/sonichack.jpg" class="someone helper"></a>
-          <h3>SonicHack</h3>
-          <span class="tag is-warning">Helper</span>
-        </div>
-		
-		<div class="column">
-          <a href="https://youtube.com/annsquake" target="_blank" rel="noopener noreferrer"><img src="/img/people/annsquake.jpg" class="someone helper"></a>
-          <h3>ANNSQuake</h3>
-          <span class="tag is-warning">Helper</span>
+    <div id="content">
+        <div id="content-inner">
+            <div class="box" id="about" style="text-align: center;">
+                <h1 class="title"><strong>About</strong></h1>
+                <h2 class="subtitle">What is Aurora?</h2>
+                <p>Aurora is a Discord community.</p>
+                <p>We're mostly interested in technology and outing tech support scammers.</p>
+            </div>
+            <div class="box" id="staff" style="text-align: center;">
+                <h1 class="title"><strong>Staff</strong></h1>
+                <h2 class="subtitle">Who's in charge?</h2>
+                <div class="columns">
+					{% comment %} Make the CEOs not so big. {% endcomment %}<div class="column"></div>
+                    <div class="column">
+                        <a href="https://thehaden.co" rel="noopener noreferrer" target="_blank"><img class="someone ceo" src="/img/people/haden.jpg"></a>
+                        <h3>Haden</h3><span class="tag is-ceo">CEO</span>
+                    </div>
+                    <div class="column">
+                        <a href="https://www.youtube.com/channel/UCi6p-LPSRkcMhEIJ2Qe0ABg/" rel="noopener noreferrer" target="_blank"><img class="someone ceo" src="/img/people/amy.jpg"></a>
+                        <h3>AtotheK</h3><span class="tag is-ceo">CEO</span>
+                    </div>
+                    <div class="column">
+                        <a href="https://dnomaid.co.uk" rel="noopener noreferrer" target="_blank"><img class="someone ceo" src="/img/people/dnomaid.jpg"></a>
+                        <h3>dnomaid</h3><span class="tag is-ceo">CEO</span>
+                    </div>
+                   	{% comment %} Make the CEOs not so big. {% endcomment %} <div class="column"></div>
```

Comment:
What's with the comments?

Reply:
so people know what the bloody blank <divs> are for if they wanna edit it

Reply:
I don't really think that it's necessary.

Reply:
Just accept it please dad.
Tom and others might be confused if they see that and remove it.

Reply:
We don't really comment the rest of our code, so

Reply:
it's fiiine